website/client/src/libs/store/helpers/filterTasks.js

```diff
@@ -23,7 +23,7 @@ const taskFilters = {
     label: 'todos',
     filters: [
       { label: 'remaining', filterFn: t => !t.completed, default: true }, // active
-      { label: 'scheduled', filterFn: t => !t.completed && t.date, sort: t => t.date },
+      { label: 'scheduled', filterFn: t => !t.completed && t.date && (!t.startShowing || new Date(t.startShowing) <= new Date()), sort: t => t.date },
```

Comment:
I tried to set `startShowing` to yesterday/today/tomorrow but with none of those values I couldn't get it to show with this filter - so there might something wrong here